js/script.js

```diff
@@ -100,12 +103,7 @@ function initializeBoard() {
 function activateBoard() {
 	logger.log("Entered into activateBoard");
 
-	for (var row = 1; row <= 3; row++) {
-		for (var col = 1; col <= 3; col++) {
-			logger.log("Adding listener for row[" + row + "] col[" + col + "]");	
-			document.getElementById("cell-" + row + col).addEventListener("click", playerMove, false);
-		}
-	}
+	addCellListeners();
```

Comment:
Hi @Glinkis , I think it's better to rename the function to "activateGrid" or "activateCells" as it goes well with the parent function naming

Reply:
Agreed.